src/main/resources/templates/db_scripts/schema.json

```diff
@@ -80,6 +80,11 @@
       "run": "before",
       "snippetPath": "data-migration/12.0.0/delete_order_invoices_relation_duplicates.ftl",
       "fromModuleVersion": "mod-orders-storage-12.1.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "data-migration/13.0.0/restriction_for_fund_codes.sql",
```

Comment:
Please move script into data-migration/13.1.0
Thanks

Reply:
It looks like migration script directories use the convention major.0.0, as opposed to using all the versions. For instance:
```    {
      "run": "after",
      "snippetPath": "data-migration/12.0.0/po_line_sync_fund_code.sql",
      "fromModuleVersion": "mod-orders-storage-12.1.0"
    },
```
It would have been better to name them simply "12" and "13", but I think it makes sense to limit the number of directories and use only major versions.
If you prefer, I could change the organization and use precise versions for all directories.

Reply:
There are 2 types version of the release : minor(13.1.0) and major (13.0.0).
It means that release can be minor or major and only one folder needs per release. 
No need to reorganize previously created migration folders